Multisig/UI/Assets/AssetsViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AssetsViewModel.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 13.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+struct TokenBalance: Identifiable {
+    var id: String {
+        return address
```

Comment:
`return` can and should be removed to use implicit returns - more straightforward.

Reply:
done